src/tools/info/build_info.c

```diff
@@ -15,8 +15,9 @@
 
 void print_version()
 {
-    printf("# UCT version=%s revision %s\n", UCT_VERNO_STRING, UCT_SCM_VERSION);
-    printf("# configured with: %s\n", UCX_CONFIGURE_FLAGS);
+    printf("# Version %s\n", UCT_VERNO_STRING);
+    printf("# SCM branch '%s' revision %s\n", UCT_SCM_BRANCH, UCT_SCM_VERSION);
```

Comment:
What SCM stands for ?

Reply:
SCM = source code management

Reply:
@shamisp fixed
```
2022-01-06T09:04:33.6364536Z + ./src/tools/info/ucx_info -s -f -c -v -y -d -b -p -w -e -uart
2022-01-06T09:04:33.6786515Z # Version 1.13.0
2022-01-06T09:04:33.6788326Z # Git branch '', revision 607b97e
2022-01-06T09:04:33.6790393Z # Configured with: --enable-gtest --enable-examples --with-valgrind --enable-profiling --enable-frame-pointer --enable-stats --enable-fault-injection --enable-debug-data --enable-mt --prefix=/home/swx-azure-svc_azpcontainer/34543-20220106.7/build/install --enable-debug --enable-examples
...
```

Reply:
@yosefe Why git branch is `''` ? I should be master or one of the branches, right ?

Reply:
if branch name is a valid English word it will separate it from the rest of the sentence for better clarity

Reply:
I think it's related to how Azure checks out the code during CI
In a manual test, it is the real branch name

Reply:
manual build:
```
$ ./build-devel/src/tools/info/ucx_info -v
 Version 1.13.0
# Git branch 'topic/ucx-info-add-branch', revision f7b2488
# Configured with: --enable-gtest --enable-examples --with-valgrind --enable-profiling --enable-frame-pointer --enable-stats --enable-fault-injection --enable-debug-data --enable-mt --prefix=/hpc/newhome/yosefe/workspace/hpc/ucx/build-devel/install --with-mpi=/labhome/yosefe/workspace/ompi-release/build-gcc/install --no-create --no-recursion
```

Reply:
Ah, AZP is probably using "anonymous" branch. Looks good